ui/blocks/PastCheckin/PastCheckin.tsx

```diff
@@ -19,7 +23,9 @@ export const PastCheckin: React.FC<Props> = ({ checkins }) => {
       {checkins.length > 1 && (
         <>
           <Box height={1} />
-          <Text variant="h4">{checkins.length} Personen</Text>
+          <Text variant="h4">
+            {checkins.length} {t('people')}
```

Comment:
This is not important for now, but to be 100% language agnostic it would be better to have the number as part of the translation string. Some languages have rather complex logic for pluralisation of words and the order between number and word.

Reply:
Easy to fix though. I'll add `people_one` and `people_many` for it.

Reply:
@bitboxer definitely. Counts and formals have to be better. I could create a separate PR for this